### PR TITLE
[WIP] Slides for CivicTechTO April 2019

### PR DESCRIPTION
**Pull request by dasanchez**
This is the draft for the slide deck I'm using at Civic Tech Toronto on April 16.
Can I have some feedback please? I've been working in a bubble for a couple of weeks now :) 

- It looks like I can just open `index.html` in Firefox without using a web server, so that's neat. 
- Here's a [recorded preview](https://ipfs.io/ipfs/QmPqouhxBJGLAJjjz6VG8NB6VNJzHatzNYd3HiZQ7rJMMk) - patcon said keep it under 10 minutes, so I had to time myself...I figured I might as well share the latest take.
- Thanks @benhylau for the initial comments!

### CODE REVIEW

**Comment by Shrinks99** [comment]
Nice presentation and good thinking to record it to get feedback.  A few notes:

1. I think you should disable the slide counter.  Because your slides include sub-slides and you go through them at a rapid pace it's not particularly indicative of the amount of time that will be taken up or even the length of the presentation, also it detracts from the actual content.  This is easy to do, in the `var slideshow = remark.create({` section (I like to put it right below the aspect ratio) stick `slideNumberFormat:'',` in there.

2. There is too much text on the Mesh Network slide.  I know you have the TL;DR on there and that's sort of funny but people will try to read it and generally people have a hard time reading and listening at the same time.

3. Presenter mode!  That's right, if you didn't know about it Remark has a presenter mode and it's great!  Hit 'C' on the keyboard to pop out a new window that is linked with your other one (both will change when you hit the space bar) and then 'P' in the main window to enable it.  Comes with a clock and a notes area and you can see the next slide!  Cool stuff.

4. Pls no Gill Sans.  Gill Sans is a wacky all over the place font and I don't like it :P  ...And it was made by [a pedophile who ALSO abused his dogs](https://en.wikipedia.org/wiki/Eric_Gill#Personal_life) if that wasn't enough so there's today's fun fact I guess.  While it still remains an important historical typeface as it's the first real humanist sans-serif I like Work Sans better (and we use it in other places too!).  The letterforms have a more consistent stroke width and generally look cleaner.  Who knew typefaces could be so political?!

File: presentations/20190416_civictechto/css/style.css [hunk 1]
```diff
@@ -0,0 +1,159 @@
+body {
+  font-family: 'Gill Sans', sans-serif;
```
Comment (Shrinks99) [inline presentations/20190416_civictechto/css/style.css]: I'm not a Gill Sans fan.  Lets replace this with something sans-Gill!

**Review by Shrinks99 — APPROVED** [review]

**Comment by Shrinks99** [comment]
Changes are all good, bonus points for introducing a thicker weight (`font-weight: bold;` for h1-h4) for titles VS body text but as it is you're good to go.

**Comment by makew0rld** [comment]
Looks great! Sometimes the images are a bit small, so watch out for that, there's more space to fill on the slides. Also Faith Goldy is likely a poor example of censorship, since she won't garner much sympathy, and was promoting hate speech. I would choose a different example, but I know that is perhaps a contentious issue since she was censored. 

Also I like the quote at the end. If you want another, I've always thought selections from the [A Declaration of the Independence of Cyberspace](https://www.eff.org/cyberspace-independence) provide a good example of what the Internet was dreamed to be, and what I want it to become again.

**Comment by dasanchez** [comment]
Thanks for the feedback so far!

@makeworld-the-better-one Ihe question I wanted to bring up with that censorship slide was why are these companies so concerned with blocking nudies but yet allow more damaging content to be distributed through their platforms...I'll give it another pass ;)

**Comment by makew0rld** [comment]
Ah, that's a good point. At the moment it seems like it's more about how Faith Goldy shouldn't be banned, which won't be well received.

**Pull request merged**